sktime/transformers/series_as_features/derivative.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import pandas as pd
+from sktime.transformers.series_as_features.base \
+    import BaseSeriesAsFeaturesTransformer
+from sktime.utils.data_container import tabularize
+from sktime.utils.validation.series_as_features import check_X
+
+
+class DerivativeTransformer(BaseSeriesAsFeaturesTransformer):
```

Comment:
Is this different from the [`DerivativeSlopeTransformer`](https://github.com/alan-turing-institute/sktime/blob/67c56be8b1e838f2628df829946f795b7dba9aed/sktime/transformers/series_as_features/summarize/_extract.py#L101), if so would go good to mention in in the init under `See also`

Reply:
Yeah it is different from DerivativeSlopeTransformer the way its calculating the Derivatives is slightly different. Which init is it?

Reply:
Um yes this is the same so I'll remove DerivativeTransformer :)